### PR TITLE
Update phpunit

### PR DESCRIPTION
**Pull request by lordrhodos**
Update `phpunit/phpunit` to version 7.5.x

### CODE REVIEW

**Review by felixsand — APPROVED** [review]
Excellent! 🎉

**Comment by felixsand** [comment]
"On hold" until final ok to merge from author

**Comment by lordrhodos** [comment]
Ok, from my side!.

**Pull request merged**
